FormData.js

```diff
@@ -386,5 +387,15 @@ if (typeof Blob === 'function' && (typeof FormData === 'undefined' || !FormData.
     }
   }
 
+  // Patch navigator.sendBeacon to use the underlying Blob
+  if (_sendBeacon) {
+    global.navigator.sendBeacon = function (url, data) {
+      if (data instanceof FormDataPolyfill) {
+        data = data['_blob']()
+      }
+      return _sendBeacon(url, data)
```

Comment:
figured out i can't accept this PR...

you need to use `call` when saving sendBeacon to some variable

```suggestion
      return _sendBeacon.call(navigator, url, data)
```
Otherwise you get `Uncaught TypeError: Illegal invocation`

Reply:
Thanks, good catch! Fixed!